app/models/city.rb

```diff
@@ -0,0 +1,5 @@
+class City < ApplicationRecord
+  has_many :businesses
+  validates :name, presence: true, uniqueness: true
```

Comment:
Here, the name and a zip code would make a City truly unique. There are many Springfield's for example, but only one in each zipcode or state.